playground/backend/cmd/server/server.go

```diff
@@ -36,6 +36,9 @@ func runServer() error {
 	if err != nil {
 		return err
 	}
+
+	logger.SetupLogger(ctx, envService.ApplicationEnvs.LaunchSite(), envService.ApplicationEnvs.GoogleProjectId())
```

Comment:
will this always expect to run on GCP? What happens if we run a test instance locally? Wil it just log locally? Or what would happen?

Reply:
ah this is answered further down. Thanks!